sleep-for-days.md

```diff
@@ -0,0 +1,81 @@
+### Go
+---
+```
+// Send an email every 30 days
+func SleepForDaysWorkflow(ctx workflow.Context) (string, error) {
+  for {
+    workflow.ExecuteActivity(ctx, SendEmail) // Activities have timeouts, and will be retried by default!
+    workflow.Sleep(ctx, time.Hour*24*30) // Sleep for 30 days!
+  }
+  // ...
+}
+```
+
+### Java
+---
+```
+public class SleepForDaysImpl implements SleepForDaysWorkflow {
```

Comment:
```suggestion
public class SleepForDaysWorkflowImpl implements SleepForDaysWorkflow {
```

Clearer to make "Impl" just be a suffix of what's impl'd IMO

Reply:
Agree